external-crates/move/crates/move-compiler/src/editions/mod.rs

```diff
@@ -187,11 +196,7 @@ impl Edition {
     fn unknown_edition_error(&self) -> anyhow::Error {
         anyhow::anyhow!(
             "Unsupported edition \"{self}\". Current supported editions include: {}",
-            Self::VALID
-                .iter()
-                .map(|e| format!("\"{}\"", e))
-                .collect::<Vec<_>>()
-                .join(", ")
+            format_oxford_list!("and", "\"{}\"", Self::VALID)
```

Comment:
Neat! Should the entries go first though? was a bit surprised seeing them at the end... but I suppose thats consistent with `format` but not with `join`?

No opinions or asks here, just took me by surprise at first and wanted to log as much

Reply:
Yes, the idea is to be consistent with format, since it's a macro invocation.